roles/epel_repositories/tasks/main.yml

```diff
@@ -6,3 +6,14 @@
   tags:
     - packages
   when: ansible_os_family == 'RedHat'
+
+- name: 'Enable CentOS-CR Repository'
+  ini_file:
+    path: /etc/yum.repos.d/CentOS-CR.repo
+    section: cr
+    option: enabled
+    value: 1
+    no_extra_spaces: True
+  tags:
+    - packages
+  when: ansible_os_family == 'RedHat'
```

Comment:
Won't this break for users who use RHEL?

Reply:
:woman_shrugging:

Reply:
fixed